packages/core/src/auth/utils.ts

```diff
@@ -800,3 +801,9 @@ export async function getAuthType() {
     }
     return authType
 }
+
+export const clientName = once(_clientName)
+function _clientName() {
```

Comment:
Thanks for cleaning this up! Should it live in `extensionUtilities` next to the other similar functions (`productName()`, etc)?

Reply:
I was wondering that as well. On the one hand the current implementation keeps it next to auth and I'm not sure that non-auth projects have a use for this? On the other hand like you mentioned, productName() etc lives in extensionUtils. I'm fine with either tbh

Reply:
perhaps mention this new function in the docstring of productName(), and vice-versa?